libs/edit-parent-profile/feature/src/lib/edit-parent-profile.component.scss

```diff
@@ -33,71 +47,26 @@
     padding-bottom: 25px;
 }
 
-.empty-div{
-    width: 33.3%;
-    height: 100%;
-    float: right;
-}
-
-.profile-picture-wrapper{
-    width: 62%;
-    height: 40%;
-    background-color: rgba($color: #032927BF, $alpha: 0.75);
-    border-radius: 16px;
-    align-items: center;
-    justify-items: center;
-    text-align: center;
-    left: 9%;
-    margin: auto;
-    display: block;
-    margin-top: 25%;
-    margin-bottom: 40px;
-}
-
-.profile-picture {
-    width: 80%;
-    height: 100%;
-}
-
-.parent-info-small{
+.details-div{
     height: 6%;
     background-color: var(--ion-background-color);
     display:flex;
     font-family: Roboto;
     align-items: center;
-    color: #79BA9E;
+    color: black;
```

Comment:
i think we use --ion-color-dark not black, unless this had specific reason to black